frontend/src/store/modules/credentials-documents.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2020-2022 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/business-partner-agent
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  AriesCredential,
+  MyDocumentAPI,
+  Page,
+  walletService,
+} from "@/services";
+import { apiBaseUrl, axios, EventBus } from "@/main";
+import { AxiosResponse } from "axios";
+import { IStateCredentialsAndDocuments } from "@/store/state-type";
+import { CredentialTypes } from "@/constants";
+
+const state: IStateCredentialsAndDocuments = {
+  credentialList: new Array<AriesCredential>(),
+  documentList: new Array<MyDocumentAPI>(),
+};
+
+export default {
+  state,
+  getters: {
+    getCredentials: (state: IStateCredentialsAndDocuments) => {
+      return state.credentialList;
+    },
+    publicDocumentsAndCredentials: (state: IStateCredentialsAndDocuments) => {
+      return [...state.credentialList, ...state.documentList].filter(
+        (d) => d.isPublic === true
+      );
+    },
+    getOrganizationalProfile: (state: IStateCredentialsAndDocuments) => {
+      const documents = state.documentList.filter(
+        (d) => d.type === CredentialTypes.PROFILE.type
```

Comment:
CredentialType?

Reply:
CredentialTypes contains the ominous "UNKNOWN" type which is not available in CredentialType. It is used for UI only functionality which I did not get into yet. I would safe it for a later PR.